Disable tests, slower to build and would need host-fltk.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura fltk-1.3.2.orig/Makefile fltk-1.3.2/Makefile
--- fltk-1.3.2.orig/Makefile	2013-10-27 18:19:39.693208127 -0300
+++ fltk-1.3.2/Makefile	2013-10-27 18:20:08.032136885 -0300
@@ -18,7 +18,7 @@
 
 include makeinclude
 
-DIRS = $(IMAGEDIRS) src $(CAIRODIR) fluid test documentation
+DIRS = $(IMAGEDIRS) src $(CAIRODIR) fluid documentation
 
 all: makeinclude fltk-config
 	for dir in $(DIRS); do\
